src/main/java/org/eclipse/che/ide/flux/liveedit/CheFluxLiveEditExtension.java

```diff
@@ -65,9 +63,13 @@
 
 import org.eclipse.che.ide.resource.Path;
 
+import static org.eclipse.che.ide.api.notification.ReadState.READ;
+import static org.eclipse.che.ide.api.notification.StatusNotification.Status.SUCCESS;
+import static org.eclipse.che.ide.api.notification.StatusNotification.DisplayMode.FLOAT_MODE;
+
 
 @Extension(title = "Che Flux extension", version = "1.0.0")
-public class CheFluxLiveEditExtension {
+public class CheFluxLiveEditExtension implements CursorModelWithHandler, CursorActivityHandler {
```

Comment:
Handler implementations should be in separate classes in my opinion, or use anonymous classes